pkg/virt-controller/watch/clone/clone_base.go

```diff
@@ -30,6 +30,7 @@ const (
 	RestoreCreated  Event = "RestoreCreated"
 	RestoreReady    Event = "RestoreReady"
 	TargetVMCreated Event = "TargetVMCreated"
+	TargetVMReady   Event = "TargetVMCreated"
```

Comment:
the msg of the event should be Ready not Created

Reply:
done